docs/user/contacts.md

```diff
@@ -0,0 +1,40 @@
+# Contacts
+## [Bug Report](https://github.com/vdaas/vald/issues/new?assignees=&labels=type%2Fbug%2C+priority%2Fmedium%2C+team%2Fcore&template=bug_report.md&title=)
+
+A bug is a demonstrable problem that is caused by code in the repository. Bug reports are helpful for developers who maintain the Vald project.
+
+A good bug report shouldn’t leave others needing to ask you constantly for more information. So could you please try to be as detailed as possible in your bug report?
+
+Things to check before reporting the bug:
+- Check your environment
+Bug may occur depending on versions (Go, Docker, Kubernetes, NGT ..etc).
+So please match versions recommended by the Vald project. 
```

Comment:
Do we need to ask user to try the recommended version before reporting a bug?

Reply:
:memo:
This line is deleted because the user does not have to match the recommended version.